src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfile.java

```diff
@@ -36,6 +39,8 @@ public class PerformanceProfile {
     @SerializedName("slo")
     private SloInfo sloInfo;
 
+    private Map<String, Terms> terms;
```

Comment:
Please double-check whether the perfprofile is associated with Terms. If it's not related, then it doesn't make sense to have a reference in the perf profile.